cli/cmd/install.go

```diff
@@ -38,6 +38,7 @@ var (
 	userInputIgnoredNamespaces []string
 	userInputIgnoredContainers []string
 	userInputInstallProfiles   []string
+	uiMode                     string
```

Comment:
suggestion: consider making it an enum in `common` module.
can also be added later.